inlong-sort/sort-single-tenant/pom.xml

```diff
@@ -166,6 +166,18 @@
             <artifactId>hadoop-minicluster</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-connector-mysql-cdc</artifactId>
+            <version>2.0.2</version>
```

Comment:
Please move the dependent version number to the pom file in the root directory.

Reply:
done